app/router.js

```diff
@@ -23,6 +23,7 @@ Router.map(function() {
     this.route('setup-instructions');
   });
   this.route('get-token');
+  this.route('user-search');
```

Comment:
Where is this route used?

Reply:
That was a route I added to easily test the component. I meant to revert the commit that added it but forgot to